.env.example

```diff
@@ -1,4 +1,5 @@
 ES_URL=http://localhost:9200
+NEO4J_URL="0.0.0.0"
```

Comment:
Why `" "`?

Reply:
Had trouble with the IP earlier but I can try removing them and see if it still works.